packages/@okta/vuepress-site/docs/guides/oin-oidc-protocols/main/index.md

```diff
@@ -8,11 +8,11 @@ This guide provides background information on the protocol requirements for OIDC
 
 ## Authorization Code flow
 
-For OIDC applications destined for the OIN, Okta recommends building a "web" application with a dedicated server-side back end that is capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
+For OIDC applications destined for the OIN, Okta recommends building a web application with a dedicated server-side back end that is capable of securely storing a client secret and exchanging information with an authorization server through trusted back-channel connections.
```

Comment:
For "back end", I've seen it as:
back end
back-end << this one seems to be used more, maybe use this one.
backend

Reply:
I would normally avoid using "back end", but don't want to replace it with a word without vetting by Eng. 
Microsoft says to use: Two words as a noun. Hyphenate as an adjective.

Reply:
👍